index.md

```diff
@@ -77,27 +77,34 @@ OLCF, NERSC, and the Swiss National Supercomputing Centre (CSCS).
 <tr>
 <td>David Zhang</td>
 <td>Seshu Yamajala</td>
-<td>Wei Wu</td>
```

Comment:
We do need to update this file, but let's split that into a different PR.

Reply:
Reverted.

Reply:
I still see changes in this file in the diff view?

Reply:
Did you forget to push the branch?

Reply:
The last commit reverted the file back to the original. Under files changed I no longer see index.md.